frontend/src/app/modules/auth/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="sign-up">
+    <form name="authForm" #authForm="ngForm" class="registration-form">
+        <h1>Sign In</h1>
```

Comment:
Ideally to make it as a general component, to which, based on route, we will set either login or sign up form

Reply:
fixed